src/frontend/js/widgets/Dashboard/components/DashboardItem/Enrollment/DashboardItemEnrollment.tsx

```diff
@@ -12,13 +13,26 @@ export const DashboardItemEnrollment = ({ enrollment }: DashboardItemCourseRunPr
     throw new Error("Enrollment's course_run must provide course attribute");
   }
 
-  return (
-    <DashboardItem
-      title={course.title}
-      code={'Ref. ' + course.code}
-      footer={
-        <DashboardItemCourseEnrolling course={course} activeEnrollment={enrollment} icon={true} />
-      }
-    />
-  );
+  const footer = [
+    <DashboardItemCourseEnrolling
+      key={`${enrollment.id}`}
+      course={course}
+      activeEnrollment={enrollment}
+      icon={true}
+    />,
+  ];
+
+  enrollment.products.forEach((product) => {
+    if (product.type === ProductType.CERTIFICATE) {
+      footer.push(
+        <ProductCertificateFooter
+          key={[enrollment.id, product.id].join('_')}
+          product={product}
+          course={course}
+        />,
+      );
+    }
+  });
```

Comment:
Not really reactish. I wonder how React manages render optimization with this logic as footer is redefined at each render.